draft-irtf-cfrg-aegis-aead.md

```diff
@@ -1611,9 +1611,11 @@ Outputs:
 Steps:
 
 ~~~
-stream, tag = Encrypt(ZeroPad({}, len), {}, key, nonce)
-
-return stream
+if len = 0:
+    return {}
+else:
+    stream, tag = Encrypt(ZeroPad({ 0 }, len), {}, key, nonce)
```

Comment:
Should we specify `0` in hex?

Reply:
`0` is a bit, not a byte, so I think we shouldn't use a hex representation here.